src/prisma/schema.prisma

```diff
@@ -2097,3 +2098,13 @@ model ReferralCodeUseEvent {
   @@index([referrerId])
   @@index([refereeId])
 }
+
+model ExternalProjects {
+  id            String  @id @default(uuid()) @db.Uuid
+  metadata      Json
+  recipient     String
+  schemaId      String?
+  // Fields specific to ceramic
+  ceramicId     String? @unique
+  ceramicRecord Json?
```

Comment:
not sure this field is necessary. this table's purpose is just to preload projects